sdk/src/beta9/vendor/nest_asyncio.py

```diff
@@ -0,0 +1,247 @@
+# BSD 2-Clause License
```

Comment:
Added the license since it seems it may need to be mentioned. 

https://github.com/erdewit/nest_asyncio/blob/master/LICENSE